src/odc/odb2nc.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+
+# Find python runtime dependency paths.
+import sys
+import os
+from pathlib import Path
+prefix_lib_path = Path(__file__).absolute().parent.parent/'lib'
+prefix_bin_path = prefix_lib_path.parent/'bin'
+sys.path.append(str(prefix_lib_path))
+sys.path.append(str(prefix_lib_path/'python{}.{}'.format(*sys.version_info)/'site-packages'))
+os.environ['PATH'] = "%s:%s" % (os.environ['PATH'], str(prefix_bin_path))
```

Comment:
I think we can remove all this now that pyodc and codc are installed on the system.